lib/r10k/api/modules.rb

```diff
@@ -123,12 +125,19 @@ def write_module(module_name, env_map, path, opts={})
             git.clean(path, git_dir: git_dir, force: true)
           end
         when :forge
-          tarball_cachedir = cachedir_for_forge_module(mod[:source], opts[:cachedir])
+          release_slug = release_slug_from_module_slug_version(mod[:source], mod[:resolved_version])
+          cachedir = opts[:cachedir]
 
-          # TODO: cache tarball
+          unpack_tmpdir = cachedir_for_forge_module([release_slug, 'unpack'].join('-'), cachedir)
+
+          release_tarball = get_cached_forge_release(release_slug, cachedir, false, opts[:forge])
+
+          begin
```

Comment:
This `begin/end` block seems to be meaningless - there's no ensure or rescue statement so it doesn't seem to do anything. Am I missing something?